qemu/tests/driver_load.py

```diff
@@ -44,15 +45,28 @@ def run(test, params, env):
     driver_id = driver_id[0]
     if params["os_type"] == "windows":
         driver_id = '^&'.join(driver_id.split('&'))
+        devcon = params.get("devcon")
+        devcon_cli = devcon % utils_misc.get_winutils_vol(session)
+        session.cmd(devcon_cli)
     driver_load_cmd = driver_load_cmd.replace("DRIVER_ID", driver_id)
     driver_unload_cmd = driver_unload_cmd.replace("DRIVER_ID", driver_id)
 
     for repeat in range(0, int(params.get("repeats", "1"))):
         error.context("Unload and load the driver. Round %s" % repeat,
                       logging.info)
-        session.cmd(driver_unload_cmd)
+        error.context("unload CLI %s" % driver_unload_cmd,
+                      logging.info)
+        status, output = session.cmd_status_output(driver_unload_cmd)
+        if params["os_type"] == "windows" and "device(s) disabled" not in output:
+            raise error.TestError("failed to unload driver %s" % output)
+        error.context("status %s unload output %s"
```

Comment:
Hi,  @suqinhuang  this line is the step instead of the output ,should use error.context